SCRWebService/endpoints/summary_care_record.py

```diff
@@ -73,6 +89,34 @@ def _extract_message_body(self):
         try:
             return json.loads(self.request.body)
         except json.decoder.JSONDecodeError as e:
-            logger.error('004', 'Exception raised whilst parsing message body: {exception}', {'exception': e})
+            logger.error('005', 'Exception raised whilst parsing message body: {exception}', {'exception': e})
             raise tornado.web.HTTPError(400, 'Failed to parse json body from request',
                                         reason=f'Exception raised while parsing message body: {str(e)}')
+
+    def _extract_correlation_id(self):
+        """
+        Attempts to extract the correlation-id from the message headers, if this fails a new one is generated,
+        either way the correlation id is set in the logger
+        :return: A UUID
+        """
+        correlation_id = self.request.headers.get('Correlation-Id', None)
+        if not correlation_id:
+            correlation_id = message_utilities.MessageUtilities.get_uuid()
+            log.correlation_id.set(correlation_id)
+            logger.info('006', "Failed to extract correlation-id from message, generated a new one")
```

Comment:
reword message, should be something like "No correlation ID provided by caller, using {ID}"

Reply:
Once set as a context var the correlation ID is added to all log lines, so this line will print out the correlation id by default